core/templates/dev/head/base_components/BaseContentDirective.ts

```diff
@@ -17,9 +17,8 @@
  */
 
 require('pages/OppiaFooterDirective.ts');
-
 require('domain/utilities/UrlInterpolationService.ts');
-
+require('domain/sidebar/SidebarStatusService.ts');
```

Comment:
Move this a line above. (In sorted order.)

Reply:
Done!